docs/en/setup/plugins/forwarder_nativeebpfprofiling-grpc-forwarder.md

```diff
@@ -0,0 +1,9 @@
+# Forwarder/nativeebpfprofiling-grpc-forwarder
```

Comment:
How about changing to `native/ebpfprofiling`? The word seems too long to read.

Reply:
Rename to `native-ebpfprofiling-grpc-forward`